stocksite/stocks/views/commitBuyView.py

```diff
@@ -0,0 +1,61 @@
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
+from accounts.models import Account
+from transactions.models import Transaction
+from .utils import MockQuoteServer, getByStockSymbol
+from time import time
+
+
+class CommitBuyView(APIView):
+    """
+    API endpoint for confirming the buy of a stock.
+    """
+    def post(self, request):
+        # Get request data
+        username = request.data.get("username")
+        # Test values (uncomment to populate the db)
+        # stockSymbol = 'NISSAN'
+        # amount = 500.00
+        
+        # Check cache for recent buy transaction (within 60 sec), if non-existent log errorEvent
+        # stockSymbol = fromcache
+        # amount = fromcache
+
+        # Find user account
+        account = Account.objects.filter(username=username).first()
+        
+        # TODO: Check for quote in cache (if not in cache/is stale perform query)
+        # Query the QuoteServer (Try/Catch for systemEvent/errorEvent logging)
+        quoteQuery = MockQuoteServer.getQuote(username, stockSymbol)
```

Comment:
Gotta be careful with MockQuoteServer here, and other places, as need to remember to change for workload testing.